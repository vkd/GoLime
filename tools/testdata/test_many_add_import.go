package test

import (
	"fmt"
	"log"

	"github.com/pkg/errors"
)

func unusedMany() string {
	fmt.Printf("format")
	log.Printf("many")
	// hello from comments
	return errors.New("unused_many").Error()
}
