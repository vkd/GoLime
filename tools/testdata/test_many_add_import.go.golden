package test

import (
	"bytes"
	"fmt"
	"log"

	"github.com/pkg/errors"
)

func unusedMany() string {
	fmt.Printf("format")
	log.Printf("many")
	// hello from comments
	return errors.New("unused_many").Error()
}
